### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by tGodson**
Jest Tests for the following functions,

>  capitalize(string) takes a string and returns that string with the first character capitalized.
>     reverseString(string) takes a string and returns it reversed.
>     A calculator object that contains the basic operations: add, subtract, divide, and multiply.
>     Caesar Cipher
>   Array Analysis. A function that takes an array of numbers and returns an object with the following properties: average, min, max, and length.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello Team 👋

**You've both done pretty well on this.**

- [x] IMPORTANT - Your pull request should have a description. Please update it to include a description of the changes made in this branch.

- [x] Kindly check other comments made on your code and fix them.

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

HAPPY CODING👨‍💻

File: package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "jquery": "^3.5.1",
+    "node-sass": "^4.14.1",
+    "popper.js": "^1.16.1",
+    "sass-loader": "^10.0.4",
+    "style-loader": "^2.0.0",
+    "url-loader": "^4.1.1",
+    "webpack": "^5.3.2",
+    "webpack-cli": "^4.1.0",
+    "webpack-dev-server": "^3.11.0",
+    "webpack-merge": "^5.3.0"
```
Comment (mikenath223) [inline package.json]: - [x] Webpack is not needed in this project please remove these dependencies.

File: src/arrayAnalysis.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-return-assign */
+const analyze = (ar) => {
+  let sum = 0;
+  let object;
```
Comment (mikenath223) [inline src/arrayAnalysis.js]: - [x] Remove this unused variable.

File: src/arrayAnalysis.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  for (let i = 0; i < ar.length; i++) {
+    sum += ar[i];
+  }
+
+  return object = {
+    average: sum / ar.length,
+    min: Math.min.apply(null, ar),
+    max: Math.max.apply(null, ar),
+    length: ar.length,
+  };
```
Comment (mikenath223) [inline src/arrayAnalysis.js]: - [x] Rather than this please simply do this instead.
```
return {
...
}
```

File: src/arrayAnalysis.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-return-assign */
+const analyze = (ar) => {
+  let sum = 0;
+  let object;
+  for (let i = 0; i < ar.length; i++) {
```
Comment (mikenath223) [inline src/arrayAnalysis.js]: - [x] Stick to eslint rule for for-loops and used this as an incrementer(i+=1).

File: src/arrayAnalysis.js [hunk 5]
```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable no-plusplus */
+/* eslint-disable no-return-assign */
```
Comment (mikenath223) [inline src/arrayAnalysis.js]: - [x] Now you can remove this eslint disabled rules.

**Review by rOluochKe — APPROVED** [review]
Hello Team
### Congratulations the project has been approved :boom: :100:
### Merge and submit the project. Good luck with the next one.
- Happy coding!

**Pull request merged**